crates/egui-winit/src/lib.rs

```diff
@@ -188,7 +188,7 @@ impl State {
     }
 
     /// Returns [`false`] or the last value that [`Window::set_ime_allowed()`] was called with, used for debouncing.
-    pub fn allow_ime(&self) -> bool {
+    pub const fn allow_ime(&self) -> bool {
```

Comment:
Is there any point in having const methods in a type that doesn't have a const constructor? No one will be able to call them in a const context anyway, and any internal refactoring might lead to breaking changes.